docs/faq.md

```diff
@@ -51,3 +51,6 @@ For convenience, we also build Docker images with `headscale`. But **please be a
 ## Why is my reverse proxy not working with Headscale?
 
 We don't know. We don't use reverse proxies with `headscale` ourselves, so we don't have any experience with them. We have [community documentation](https://headscale.net/reverse-proxy/) on how to configure various reverse proxies, and a dedicated [Discord channel](https://discord.com/channels/896711691637780480/1070619818346164324) where you can ask for help to the community.
+
+## Can I use headscale and tailscale on the same machine?
+Running headscale on a machine that is also in the tailnet might cause problems with, at least, routing and MagicDNS. It should be extremely discouraged.
```

Comment:
```suggestion
Running headscale on a machine that is also in the tailnet can cause problems with subnet routers, traffic relay nodes, and MagicDNS. It might work, but it is not supported.
```